spec/features/user_can_delete_posts_spec.rb

```diff
@@ -10,13 +10,41 @@
       password: 'password',
       password_confirmation: 'password'
     )
-    click_link 'Sign out'
-    sign_in_and_add_post(message: 'Post number one')
+    add_post(message: 'Post number one')
     add_post(message: 'This is the second post')
 
     first(:button, 'Delete').click
 
     expect(page).to have_content('Post number one')
     expect(page).not_to have_content('This is the second post')
   end
+
+  scenario 'Users are not able to delete posts not their own' do
```

Comment:
Could this be 'Users are not able to delete posts that are not their own'?

Reply:
changed..